### PR TITLE
add module mail_compose_message_default_attachment

### PR DESCRIPTION
**Pull request by kmatthes**
This module easily allows to add attachment files when composing an email.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
I see the module, but I don't see where you define the attachment to use. Is this a kind of patch?

**Comment by kmatthes** [comment]
Yeah it's a patch of mail.compose.message to add attachment_ids (given as default_attachment_ids in context) to email.

**Comment by pedrobaeza** [comment]
You should propose it then as a patch in OCB and Odoo.

File: mail_compose_message_default_attachments/mail_compose_message_default_attachments.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+#
+#   You should have received a copy of the GNU Affero General Public License
+#   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+###############################################################################
+from openerp.osv import orm
+
+
+class mail_compose_message(orm.Model):
+    """Patch mail.compose.message to add attachment_ids (given as \
```
Comment (yvaucher) [inline mail_compose_message_default_attachments/mail_compose_message_default_attachments.py]: With `"""` you don't need `\`

File: mail_compose_message_default_attachments/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+#   You should have received a copy of the GNU Affero General Public License
+#   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+###############################################################################
+
+{
+    'name': 'Compose Message Default Attachments',
+    'version': '1.0',
+    'category': 'Patch',
+    'description': "Patch mail.compose.message to add attachment_ids (given \
```
Comment (yvaucher) [inline mail_compose_message_default_attachments/__openerp__.py]: Please use `"""` for description string

**Comment by OSguard** [comment]
@kmatthes can you fix to comments

@pedrobaeza having it as module we can make it requiere for other module need this function. Having this as patch (if it will merged) we need to add a message to description 'you need this patch that this module works' but maybe we can add a patch to odoo/master

**Comment by pedrobaeza** [comment]
@OSguard, the idea is to get this into upstream and OCB branches, so you don't need to patch nothing. Maybe this won't get to both, but if you don't try, you won't know it. If not, then you can continue with this, and call the module with _fix at the end to notice what is this.

**Comment by pedrobaeza** [comment]
Closing as no answer received

**Pull request closed without merging**